### PR TITLE
Update K8s deployment to remove beta references and improve clarity

### LINKED ISSUE #208: (Feedback) Google Workspace with Kubernetes deployment
Hey team,

Thanks for tool and docs.  I've got a bit of feedback, maybe an unexplored use case or order of operations.  I'm on an Ops team and our IT came to me looking to deploy the SCIM bridge for use with Google Workspace.  I came here, looked through your Kubernetes examples and deployed pretty much the same as what's in your README here https://github.com/1Password/scim-examples/blob/master/kubernetes/README.md

I know our IT uses Google Workspace but skipped the optional section since I wasn't sure what they were planning.  I gave IT the go ahead and they got stuck [here](https://support.1password.com/scim-google-workspace/#step-2-configure-your-scim-bridge) in the user documentation where they are directed to upload the service account key.  

From the UI they get this error when trying to load the file:
read-only
  

Makes sense to me, they are trying to write to a Secret that doesn't exist.  Easy fix, I worked with them to get the details and populated the necessary Secrets using `kubectl`.  Obviously this is a bit clunky for the user and doesn't line up with the user docs so I thought I'd pass along the experience.

### PR DESCRIPTION
**Pull request by scottisloud**
This PR enhances the Google Workspace directions in the Kubernetes deployment readme. 
* It removes references to a "workspace beta"
* Links out to relevant support documents where required
* Changes the order of the secrets creation so directions to create the `workspace-credentials.json` secret immediately follow the directions to create the service account and download the credentials file. This should make the directions more straightforward. 
* Removed "Optional" from Google Workspace instructions to make it clearer that this are _required_ but only for GW customers. 
* Fixed several typos I introduced in a previous PR. 

This also includes extensive directions from @ag-adampike about custom load balancers and other more advanced use-cases.

Resolves #208.

### CODE REVIEW

File: kubernetes/README.md [hunk 1]
```diff
@@ -51,20 +51,35 @@ Once complete, create a Kubernetes Secret containing the contents of the `scimse
...
-## (Optional) Create the Google Workspace Kubernetes secrets
+## For Google Workspace customers only
 
-This section is only relevant to those enrolled in our Google Workspace provisioning beta. Contact your 1Password Sales Representative for more information on this initiative.
+This section is only relevant if Google Workspace is your identity provider. If you are not connecting 1Password to Google Workspace, please proceed to [Deploy to the Kubernetes cluster](https://github.com/1Password/scim-examples/tree/master/kubernetes#deploy-to-the-kubernetes-cluster).
 
-First, you’ll need to edit the file located at `scim-examples/beta/workspace-settings.json`, and fill in the values according to those presented in our Google Workspace documentation.
+### Create a Google service account
+
+You will need to create a Google service account and key as outlined in the 1Password support article [Connect Google Workspace to 1Password SCIM Bridge](https://support.1password.com/scim-google-workspace/#step-1-create-a-google-service-account-key-and-api-client).
```
Comment (ag-adampike) [inline kubernetes/README.md:60]: Avoid using [wants and needs](https://support.1password.com/style-guide/#wants-and-needs):

```suggestion
Create a Google service account and key as outlined in the 1Password support article [Connect Google Workspace to 1Password SCIM Bridge](https://support.1password.com/scim-google-workspace/#step-1-create-a-google-service-account-key-and-api-client). 
```

File: kubernetes/README.md [hunk 2]
```diff
@@ -51,20 +51,35 @@ Once complete, create a Kubernetes Secret containing the contents of the `scimse
...
 
 ```bash
-kubectl create secret generic workspace-settings --from-file=workspace-settings.json=/path/to/workspace-settings.json
+kubectl create secret generic workspace-credentials --from-file=workspace-credentials.json=/path/to/<keyfile>.json
 ```
 
-Then, you will need to provide the Google Service Account key file, also generated according to our documentation. Substitute `<keyfile>` with the filename generated by Google for your Google Service Account.
+### Create a Workspace settings Kubernetes secret
+
+In order to provide the SCIM bridge with necessary information about your Google Workspace account, you’ll need to edit the file located at `scim-examples/beta/workspace-settings.json`, filling in correct values for:
```
Comment (ag-adampike) [inline kubernetes/README.md:74]: We should figure out where to keep this file in a future PR, as it doesn't really belong in the `beta` folder aymore.

File: kubernetes/README.md [hunk 3]
```diff
@@ -75,14 +90,13 @@ Run the following command deploy SCIM bridge:
...
+If you are a Google Workspace customer and have stored your `workspace-credentials.json` and `workspace-settings.json` files in the same directory as your deployment files, you may see the following error after running `kubectl apply -f .`:
 
-```
+```bash
 error validating "workspace-credentials.json": error validating data: [apiVersion not set, kind not set]; if you choose to ignore these errors, turn validation off with --validate=false
 error validating "workspace-settings.json": error validating data: [apiVersion not set, kind not set]; if you choose to ignore these errors, turn validation off with --validate=false
 ```
-
-You can safely ignore these errors, or move the workspace-credentials.json and workspace-settings.json to a different directory when running `kubectl apply -f .`.
+You can safely ignore these errors, or move the `workspace-credentials.json` and `workspace-settings.json` to a different directory when running `kubectl apply -f .`.
```
Comment (ag-adampike) [inline kubernetes/README.md:99]: Little nitpick:
```suggestion
You can safely ignore these errors, or move the `workspace-credentials.json` and `workspace-settings.json` to a different directory before running `kubectl apply -f .`.
```

**Review by ag-adampike — CHANGES_REQUESTED** [review]
A couple small suggestions and someting to consider for a future update, but otherwise it's good to go.

**Comment by scottisloud** [comment]
Good suggestions. Changes made and committed!

**Review by ag-adampike — APPROVED** [review]
LGTM! 🎉

**Review by black-bryan — COMMENTED** [review]
LGTM 🎉

**Pull request merged**